common/djangoapps/status/models.py

```diff
@@ -37,7 +41,7 @@ def full_message(self, course_key):
                 course_home_message = self.coursemessage_set.get(course_key=course_key)
                 # Don't override the message if course_home_message is blank.
                 if course_home_message:
-                    msg = u"{} <br /> {}".format(msg, course_home_message.message)
+                    msg = HTML(u"{} <br /> {}").format(HTML(msg), HTML(course_home_message.message))
```

Comment:
So just to make sure I understand, these became lint errors because the linter saw that you touched this file?

Reply:
Yeah, xsslint seems configured to force fixes in any touched files at this point.